EasyPost.Tests/ServicesTests/CarrierAccountServiceTest.cs

```diff
@@ -43,6 +44,26 @@ public async Task TestCreate()
             Assert.StartsWith("ca_", carrierAccount.Id);
         }
 
+        [Fact]
+        [CrudOperations.Create]
+        [Testing.Parameters]
+        public async Task TestCreateWithAlternateEndpoint()
```

Comment:
Let's call this the suggestion below, fits better into the EasyPost language

```suggestion
        public async Task TestCreateWithCustomWorkflow()
```